addons/l10n_uy/data/account_tax_data.xml

```diff
@@ -1,31 +1,97 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+
     <!-- Account Tax Tags -->
-    <record id="tag_ventas_22" model="account.account.tag">
-        <field name="name">IVA Ventas (22%)</field>
+    <record id="tag_uy_01" model="account.account.tag">
```

Comment:
What are tag_uy_01 and tag_uy_02 used for? Did you define them just to match the first two lines of the VAT (IVA, in Spanish) report on your enterprise branch? These lines don't need tags, since they aggregate other lines.